index.js

```diff
@@ -27,4 +27,4 @@ inputText.addEventListener("input", (e) => {
 // Case TransformStream in the Text Utilities
 
 document.getElementById('lowerCased').innerText.toUpperCase()
-document.getElementById('upperCased').innerText.toLowerCase()
\ No newline at end of file
+document.getElementById('upperCased').innerText.toLowerCase()
```

Comment:
you can use css text-transform property instead for this js functionality

Reply:
Sure, will do that